### ISSUE TITLE
DOCS | Treningsstrategi | Hvordan vi har valgt å trene modellen

### ISSUE THREAD

**Issue by Johannestj** [kartverket/BirdAI#127]
### Oppgavebeskrivelse
**Arbeidsoppgave:** Skriv om treningsstrategien gruppen har valgt å bruke ved trening av modellene.
Potensielt Innhold:
- Valg av treningsparametere (batch size, learning rate, epochs) 
- Transfer learning vs. trening fra bunnen 
- Håndtering av ubalansert datasett (hvis relevant) 
- Valg av loss function og optimiseringsalgoritme 
- Treningsplan (f.eks. gradvis økning av kompleksitet) 

**Kontekst:** Viktig å vise at vi har en forklaring på hvorfor vi har valgt å trene modellene opp slik vi har gjort.

### Definition of Done (DoD)
**Krav for fullført oppgave:** Skriv en sammenhengende tekst, med innholdet over, av den kvalitet hvor den er godkjent av sekundærperson.

**Comment by martinstereo** [comment]
Skrevet litt om det viktigste av treningsstrategien, gikk ikke inn i detaljer på alt av loss function, optimaliseringsalgoritme etc. som kommr litt an på hvilken modell man velger etc.

**Comment by Johannestj** [comment]
@sanderjn1 gir tilbakemeldinger.
